docs/modules/ROOT/pages/maintenance_process.adoc

```diff
@@ -0,0 +1,78 @@
+= Maintenance Process
+
+== Overview
+This page provides information about our maintenance process.
+Basically we have two kind of maintenances procedures.
+For the APPUiO clusters and Openshift clusters our internal Squad _Sirius_ is responsible.
+The managed servers -- the more "classical" servers/services -- are updated in responsibility of our internal Squad _Polaris_.
+
+== APPUiO Public Maintenance Process
+APPUiO Public servers are updated every Tuesday with the current security and software patches provided by the Linux distributions.
+Critical security vulnerabilities in the Openshift software are announced by RedHat via https://access.redhat.com/security/security-updates/#/[Security Advisories], whose mailing list includes the VSHN Engineers.
+This security advisories are analysed within one business day.
+If a vulnerability affects the system directly, the update is planned as soon as possible.
+
+== Managed Server Maintenance Process
+All servers managed by VSHN are underlying a regular maintenance process.
+The regularly maintenance usually takes place on Tuesdays.
+On that day the current patches and updates provided by the repositories of the running operation system are installed.
+
+Even though we at VSHN like to automate every possible process, the maintenance still has some manual steps.
+In theory the updates should not break running services, but since we are managing over 1000 servers with every update something could break.
+For this reason a VSHN engineer is supervising and running the updates.
+
+VSHN agrees with every customer, when their servers could be updated.
+We do have currently 4 maintenance windows.
+Tuesday morning, Tuesday afternoon starting 13:00, Tuesday evening starting 18:00, and finally Tuesday night starting 22:00.
+Like that every customer can choose the window for their services regarding the criticality, time zones of end users, and other thoughts.
+
+Further there are services and servers, where customers chose only to update the servers once a month.
```

Comment:
(Grammar) -- Don't use commas between partial sentences (German _Hauptsatz_ and _Nebensatz_), only the [Oxford comma](https://en.wikipedia.org/wiki/Serial_comma) should be the exception.
(Grammar) -- Use temporal word at the beginning or the end of the sentence (not or rarely in the middle).
(Wording) -- Isn't "where" the Swiss "wo" (used for everything that start a dependent sentence)? Use _who_, _which_, or _that_ in doubt.
(Wording) -- I believe, "further" is a word motivated by German vocabulary. I'd suggest:

> In addition, there are services and hosts that some customers specifically choose to update once a month only.